data-prepper-plugins/kafka-source/README.md

```diff
@@ -0,0 +1,150 @@
+# Kafka source
+
+This is the Data Prepper Kafka source plugin that reads records from Kafka topic. It uses the consumer API provided by Kafka to read messages from the broker.
```

Comment:
minor comments:
".. Kafka topic" -> "Kafka topics"
".. consumer API provided by Kafka to ..". -> "consumer API provided by Kafka client library